### PR TITLE
Create 2019-03-25-coreweekly-week-12-2019.md

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

**Review by mickaelandrieu — APPROVED** [review]

File: news/_posts/core-weekly/2019-03-25-coreweekly-week-12-2019.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+* [#12641](https://github.com/PrestaShop/PrestaShop/pull/12641): Tax create/edit form migration. Thank you [@zuk3975](https://github.com/zuk3975)
+* [#12647](https://github.com/PrestaShop/PrestaShop/pull/12647): Migration of "Design -> pages -> add new / edit page category". Thank you [@tomas862](https://github.com/tomas862)
+* [#12705](https://github.com/PrestaShop/PrestaShop/pull/12705): Update OrderHistory to manage {followup} & {shipping_number} vars in all email templates, not only in in_transit template. Thank you [@jf-viguier](https://github.com/jf-viguier)
+* [#12782](https://github.com/PrestaShop/PrestaShop/pull/12782): Fixes for migrated Categories add/edit actions. Thank you [@sarjon](https://github.com/sarjon)
+* [#12897](https://github.com/PrestaShop/PrestaShop/pull/12897): Enable lite display for new theme, by [@matks](https://github.com/matks)
+* [#12946](https://github.com/PrestaShop/PrestaShop/pull/12946): Fix issue in AdminCarrierWizardController::processRanges(). Thank you [@Matt75](https://github.com/Matt75)
+* [#12971](https://github.com/PrestaShop/PrestaShop/pull/12971): Enable customers page. Thank you [@sarjon](https://github.com/sarjon)$
```
Comment (matks) [inline news/_posts/core-weekly/2019-03-25-coreweekly-week-12-2019.md:54]: There's a `$` sign at the end that is useless 😉

File: news/_posts/core-weekly/2019-03-25-coreweekly-week-12-2019.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+This is the very last week to submit a pull request and make it approved: code freeze is due on Friday, 25th of March.
+
+The work of the maintainers is currently focused on code review and merge, plus finalizing their current work. Don't forget that each morning, you can download a fresh [nighlty build](https://nightly.prestashop.com/) with the latest code merged in the PrestaShop open source project. That should help to build and test an Alpha release, hopefully next week.
```
Comment (matks) [inline news/_posts/core-weekly/2019-03-25-coreweekly-week-12-2019.md:23]: nightly, not nighlty 😉

File: news/_posts/core-weekly/2019-03-25-coreweekly-week-12-2019.md (hunk 1 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-03-25-coreweekly-week-12-2019.md:54]: done, thanks

File: news/_posts/core-weekly/2019-03-25-coreweekly-week-12-2019.md (hunk 2 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-03-25-coreweekly-week-12-2019.md:23]: done, thanks

**Pull request merged**